docs/Configuration.rst

```diff
@@ -2882,6 +2882,8 @@ explanation of properties used for eXo Platform caches.
 		-  :ref:`Cache Levels <MonitoringGadget.CacheLevels>` in User Guide.
 
 		-  :ref:`Cache management view <Management.ManagementViews.CacheManagementView>`.
+		
+You can have the full list of different caches and their values `here <https://github.com/exodev/platform/blob/develop/extension/config/src/main/resources/conf/platform/configuration.properties#L136>`__		
```

Comment:
No need to link to the internal configuration properties file IMO.
We just have to tell that the below properties are all the cache configuration properties with their default value and that they must be changed to tune eXoPlatform.

Reply:
OK